Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -83,9 +83,21 @@ void AdvancedWidget::CreateWidgets()
   utility_layout->addWidget(m_dump_textures, 2, 0);
   utility_layout->addWidget(m_dump_efb_target, 2, 1);
 
-  utility_layout->addWidget(m_use_fullres_framedumps, 3, 0);
+  // Frame dumping
+  auto* dump_box = new QGroupBox(tr("Frame Dumping"));
+  auto* dump_layout = new QGridLayout();
+  dump_box->setLayout(dump_layout);
+
+  m_use_fullres_framedumps = new GraphicsBool(tr("Dump at Internal Resolution"),
+                                              Config::GFX_INTERNAL_RESOLUTION_FRAME_DUMPS);
+  m_dump_use_ffv1 = new GraphicsBool(tr("Use Lossless Codec (FFV1)"), Config::GFX_USE_FFV1);
+  m_dump_bitrate = new GraphicsInteger(0, 1000000, Config::GFX_BITRATE_KBPS, 1000);
```

Comment:
Perhaps we shouldn't create this if `!HAVE_FFMPEG`. But the FFV1 option was created before anyway.